storage/redis_cluster.go

```diff
@@ -97,6 +97,69 @@ func NewRedisClusterPool(isCache, isAnalytics bool, conf config.Config) redis.Un
 	return client
 }
 
+func NewRedisClusterPoolWithOptions(isCache, isAnalytics bool, conf config.Config, cb func(*redis.UniversalOptions)) redis.UniversalClient {
```

Comment:
Why we can't just modify original function implementation? 
I do not think that copypaste here justified: it has only one line difference. 
If you want make this function to keep same signature, just add this last `cb` argument as optional e.g. with `...`

Reply:
I remembered an alternative implementation here while checking out code coverage, now invoke the longer constructor from the shorter one, just wrapping all the implementation. I consider this concern resolved :)